less/overrides/buttons.less

```diff
@@ -150,7 +150,9 @@
     padding: 28px 20px;
     border-radius: @border-radius-large;
   }
-
+  &.btn-round-border {
+    border-radius: 100px;
```

Comment:
another option (if you want to stick with the prior naming scheme) is call it `.btn-extra-large-radius` and create a variable called `@border-radius-extra-large` and set it to 100. like:
```
# in variables.less
@border-radius-extra-large = 100px;

# in buttons.less
.btn.btn-extra-large-radius {
  border-radius: @border-radius-extra-large;
}